tests/test_commands.py

```diff
@@ -44,7 +45,28 @@
         'iprop': 1,
         'idf': 1,
         'kcmplx': 1
-    }
+}
+
+
+def test_cmd_class():
+    OUTPUT = """This is the output.
+This is the second line.
+These are numbers 1234567890.
+These are symbols !"£$%^^@~+_@~€
+This is for the format: {format1}-{format2}-{format3}"""
+
+    # Commands options
+    cmd = '/INPUT'
+    cmd_out = CommandOutput(OUTPUT, cmd=cmd)
+
+    assert isinstance(cmd_out, (str, CommandOutput))
+    assert isinstance(cmd_out[1:], (str, CommandOutput))
+    assert isinstance(cmd_out.splitlines(), list)
+    assert isinstance(cmd_out.splitlines()[0], (str, CommandOutput))
+    assert isinstance(cmd_out.replace('a', 'c'), (str, CommandOutput))
+    assert isinstance(cmd_out.partition('g'), tuple)
+    assert isinstance(cmd_out.split('g'), list)
```

Comment:
Great tests.

Reply:
Thank you! 

Although it made more sense when we were modifying all those methods. Those test where the safeguard the modification was made as expected. I don't think they made sense anymore.